### PR TITLE
cleaning up lifts and extensions

### PR DESCRIPTION
**Pull request by EgbertRijke**

### CODE REVIEW

File: src/orthogonal-factorization-systems/lifts-of-maps.lagda.md [hunk 1]
```diff
@@ -55,25 +76,25 @@ module _
...
-    {X : UU l3} (f : X → B) → is-trunc k (lift-of i f)
-  is-trunc-lift-of k is-trunc-A is-trunc-B f =
+    {X : UU l3} (f : X → B) → is-trunc k (lift i f)
+  is-trunc-lift k is-trunc-A is-trunc-B f =
     is-trunc-Σ
       ( is-trunc-function-type k is-trunc-A)
       ( is-trunc-is-lift k is-trunc-B f)
   
-  is-trunc-lift :
+  is-trunc-total-lift :
```
Comment (fredrik-bakke) [inline src/orthogonal-factorization-systems/lifts-of-maps.lagda.md:91]: Given `compute-total-lift`, isn't this lemma obsolete?

File: src/orthogonal-factorization-systems/extensions-of-maps.lagda.md [hunk 2]
```diff
@@ -160,26 +181,26 @@ module _
...
-    is-trunc k (extension-of i P f)
-  is-trunc-extension-of k is-trunc-P f =
+    is-trunc k (extension i P f)
+  is-trunc-extension k is-trunc-P f =
     is-trunc-Σ
       ( is-trunc-Π k is-trunc-P)
       ( is-trunc-is-extension k (is-trunc-succ-is-trunc k ∘ (is-trunc-P ∘ i)) f)
 
-  is-trunc-extension :
+  is-trunc-total-extension :
```
Comment (fredrik-bakke) [inline src/orthogonal-factorization-systems/extensions-of-maps.lagda.md:197]: Isn't this obsolete due to `compute-total-extension`?

**Review by fredrik-bakke — CHANGES_REQUESTED** [review]
Thank you for this! It helps to not just have to guess at how things should be when I make additions.
While you're cleaning up, could you fix the links in the "See also" section? I forgot to change these when moving the files.

File: src/orthogonal-factorization-systems/lifts-of-maps.lagda.md (hunk 1 above)
Comment (fredrik-bakke) [reply src/orthogonal-factorization-systems/lifts-of-maps.lagda.md:91]: At the very least we can strengthen the statement.

File: src/orthogonal-factorization-systems/extensions-of-maps.lagda.md (hunk 2 above)
Comment (fredrik-bakke) [reply src/orthogonal-factorization-systems/extensions-of-maps.lagda.md:197]: Or I guess at least the proof can be simplified.

**Comment by EgbertRijke** [comment]
The lemmas about truncation levels aren't obsolete, but they can be simplified. Thanks for catching this!

**Comment by fredrik-bakke** [comment]
Looks good now, thanks again!

**Pull request merged**